src/sentry/utils/email/address.py

```diff
@@ -54,3 +59,8 @@ def domain_from_email(email):
 
 def is_valid_email_address(value: str) -> bool:
     return not settings.INVALID_EMAIL_ADDRESS_PATTERN.search(value)
+
+
+def parse_email(email: str) -> str:
+    # TODO(mgaeta): This is too brittle and doesn't pass types.
```

Comment:
Does this comment still stand? I noticed you checked the whole directory into mypy so I assume in this PR it has to pass types, right?